app/controllers/App.scala

```diff
@@ -41,7 +41,15 @@ object App extends Controller with PanDomainAuthActions {
         }
       }
 
-      val clientConfig = ClientConfig(Config().capiUrl, Config().capiKey, allTags, permittedTags.toList, permissions, "/reauth")
+      val clientConfig = ClientConfig(
+        capiUrl = Config().capiUrl,
+        capiPreviewUrl = "/support/previewCapi",
+        capiKey = Config().capiKey,
```

Comment:
Why do you need to return the capiUrl and capiKey to the client if you are going via the proxy?